DX11Renderer/MatrixAnimation/src/Renderer.cpp

```diff
@@ -12,7 +13,7 @@ bool Renderer::Initialize() {
     if (!WinApp::Initialize())
         return false;
 
-    D3DUtils::CreateTexture("free_texture2.jpg", m_texture, m_textureResourceView, m_device);
+    D3DUtils::CreateTexture("assets/textures/free_texture1.jpg", m_texture, m_textureResourceView, m_device);
```

Comment:
Path management needs more attention. Consider using `std::filesystem`. Opening a new issue for this.